en/publish/publish-wechatgame.md

```diff
@@ -68,23 +68,47 @@ In addition, the problem of file saving failure due to cache space exceeding the
 
 At the same time, when the md5Cache function of the engine is enabled, the URL of the file will change with the content of the file, so that when a new version of the game is released, the resources of the old version will naturally fail in the cache, and you can only request new resources from the server, which also achieves the effect of version control.
 
-Specifically, developers need to do:
+Starting with v2.4.0, all resources are managed in asset bundles. The engine has some builtin bundles:
+- main: the main bundle of the game, minus all other asset bundle resources, the rest of the resources belong to the main bundle
+- resources: resources in `assets://resources`
+- internal: resources in `internal://resources`
 
-1. When building, check the **MD5 Cache** in the **Build** panel.
-2. Set **Remote Service Address**, and then click **Build** button.
-3. When the build is complete, upload the **res** folder in the mini game release package to the server.
-4. Delete the **res** folder inside the release package.
-5. For the test phase, you may not be able to deploy to the official server, you need to use the local server to test, then open the **Details** page in the WeChat DevTools, check the **Does not verify valid domain names, web-view (business domain names), TLS versions and HTTPS certificates** option in the project settings.
+If you need to upload the resource onto your server, configure the asset bundle where the resource is located as a remote bundle. Next, let's take a look at the specific operational steps:
```

Comment:
If you need to upload the resource to the server, configure the `asset bundle` where the resource is located as a `remote bundle`. The operation steps are as follows: